src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -94,6 +94,31 @@ public void add(Person toAdd) throws DuplicatePersonException {
         internalList.add(toAdd);
     }
 
+
+    /**
+     * Sorts the list in alphabetical order according to the person's name.
+     */
+    public List<Person> sort(){
+        List<Person> sortedList = internalList;
+        Collections.sort(sortedList, new Comparator<Person>() {
+            @Override
+            public int compare(Person firstPerson, Person secondPerson) {
+                return firstPerson.getName().fullName.compareTo(secondPerson.getName().fullName);
+            }
```

Comment:
Take note that you should avoid doing such method calls to have better code readability.